src/service/extensions-impl.js

```diff
@@ -452,7 +464,9 @@ export class Extensions {
     // Install runtime styles.
     installStylesForDoc(
       ampdoc,
-      isExperimentOn(this.win, 'fie-css-cleanup')
+      isExperimentOn(this.win, 'fie-css-cleanup') &&
```

Comment:
It seems like this should be an option into `installExtensionsInFie`, not something that `installExtensionsInFie` itself checks. That'll move the size increase to A4A.

Reply:
yep, this is done here: https://github.com/ampproject/amphtml/pull/23493

Reply:
Size delta is negligible now.